<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string/>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_filter">
         <property name="text">
          <string>Filte</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_deviceList" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>39</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>     Capture Interface</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="DisplayInterfaces" name="listWidget"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="DisplayTable" name="tableWidget">
       <property name="font">
        <font>
         <family>Ubuntu</family>
        </font>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustIgnored</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>No.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Source</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Destination</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Info</string>
        </property>
       </column>
      </widget>
      <widget class="DisplayTree" name="treeWidget">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="DisplayRawData" name="rawDataPanel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::NoPen</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>0</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>0</number>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderMinimumSectionSize">
        <number>0</number>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDissect">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionDissector_options"/>
   </widget>
   <widget class="QMenu" name="menuCpature">
    <property name="title">
     <string>Cpature</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionRestart"/>
   </widget>
   <widget class="QMenu" name="menuStatics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionCapture_file_properitys"/>
    <addaction name="actionConversations"/>
    <addaction name="actionactionEndpoints"/>
    <addaction name="actionProtocolHierarchy"/>
    <addaction name="actionPacketLengths"/>
    <addaction name="actionIOGraph"/>
   </widget>
   <addaction name="menuStatistics"/>
   <addaction name="menuDissect"/>
   <addaction name="menuCpature"/>
   <addaction name="menuStatics"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>26</width>
     <height>26</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionRestart"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionClose"/>
   <addaction name="separator"/>
   <addaction name="actionScrollToLastLine"/>
   <addaction name="separator"/>
   <addaction name="actionEnlargeTextSize"/>
   <addaction name="actionShrinkTextSize"/>
   <addaction name="actionDefaultTextSize"/>
   <addaction name="actionResizeTableWidgetTOFitContents"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resource/mysharkres.qrc">
     <normaloff>:/icos/start.ico</normaloff>
     <normalon>:/icos/start.ico</normalon>:/icos/start.ico</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start capture packets</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset>
     <normalon>:/icos/stop.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop capture packets</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset>
     <normalon>:/icos/restart.ico</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
   <property name="toolTip">
    <string>Restart current capture</string>
   </property>
  </action>
  <action name="actionEnlargeTextSize">
   <property name="icon">
    <iconset resource="resource/mysharkres.qrc">
     <normaloff>:/icos/enlargeTextSize.ico</normaloff>:/icos/enlargeTextSize.ico</iconset>
   </property>
   <property name="text">
    <string>EnlargeTextSize</string>
   </property>
   <property name="toolTip">
    <string>Enalrge the main window text size</string>
   </property>
  </action>
  <action name="actionShrinkTextSize">
   <property name="icon">
    <iconset resource="resource/mysharkres.qrc">
     <normaloff>:/icos/shrinkTextSize.ico</normaloff>:/icos/shrinkTextSize.ico</iconset>
   </property>
   <property name="text">
    <string>ShrinkTextSize</string>
   </property>
   <property name="toolTip">
    <string>Shrink the main window text size</string>
   </property>
  </action>
  <action name="actionDefaultTextSize">
   <property name="icon">
    <iconset resource="resource/mysharkres.qrc">
     <normaloff>:/icos/defaultTextSize.ico</normaloff>:/icos/defaultTextSize.ico</iconset>
   </property>
   <property name="text">
    <string>DefaultTextSize</string>
   </property>
   <property name="toolTip">
    <string>Change the main window text size to default</string>
   </property>
  </action>
  <action name="actionScrollToLastLine">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource/mysharkres.qrc">
     <normaloff>:/icos/scroll_to_bottom.ico</normaloff>:/icos/scroll_to_bottom.ico</iconset>
   </property>
   <property name="text">
    <string>ScrollToLastLine</string>
   </property>
   <property name="toolTip">
    <string>Scroll to last line</string>
   </property>
  </action>
  <action name="actionResizeTableWidgetTOFitContents">
   <property name="icon">
    <iconset resource="resource/mysharkres.qrc">
     <normaloff>:/icos/fit_content.ico</normaloff>:/icos/fit_content.ico</iconset>
   </property>
   <property name="text">
    <string>ResizeTableWidgetTOFitContents</string>
   </property>
   <property name="toolTip">
    <string>Resize packets list columns to fit contents</string>
   </property>
  </action>
  <action name="actionDissector_options">
   <property name="text">
    <string>Dissectors options</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionCapture_file_properitys">
   <property name="text">
    <string>Capture file properties</string>
   </property>
  </action>
  <action name="actionConversations">
   <property name="text">
    <string>Conversations</string>
   </property>
  </action>
  <action name="actionactionEndpoints">
   <property name="text">
    <string>Endpoints</string>
   </property>
  </action>
  <action name="actionPacketLengths">
   <property name="text">
    <string>PacketLengths</string>
   </property>
  </action>
  <action name="actionProtocolHierarchy">
   <property name="text">
    <string>Protocol Hierarchy</string>
   </property>
  </action>
  <action name="actionIOGraph">
   <property name="text">
    <string>IO Graph</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayTable</class>
   <extends>QTableWidget</extends>
   <header>./ui/maindisplaywidget/displaytable.h</header>
  </customwidget>
  <customwidget>
   <class>DisplayTree</class>
   <extends>QTreeWidget</extends>
   <header>./ui/maindisplaywidget/displaytree.h</header>
  </customwidget>
  <customwidget>
   <class>DisplayRawData</class>
   <extends>QTableWidget</extends>
   <header>./ui/maindisplaywidget/displayrawdata.h</header>
  </customwidget>
  <customwidget>
   <class>DisplayInterfaces</class>
   <extends>QListWidget</extends>
   <header>./ui/maindisplaywidget/displayinterfaces.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource/mysharkres.qrc"/>
 </resources>
 <connections/>
</ui>
